src/coprocessor/endpoint.rs

```diff
@@ -537,8 +570,7 @@ fn err_resp(e: Error, metrics: &mut BasicLocalMetrics) -> Response {
             "lock"
         }
         Error::Outdated(deadline, now, scan_tag) => {
-            let elapsed =
-                now.duration_since(deadline) + Duration::from_secs(REQUEST_MAX_HANDLE_SECS);
+            let elapsed = now.duration_since(deadline) + Duration::from_secs(request_timeout_secs);
```

Comment:
What's this formula mean, why do we need to add the request timeout here?

Reply:
@BusyJay

Reply:
the `request_timeout_secs` is the same as REQUEST_MAX_HANDLE_SECS

Reply:
We are measuring the time elapsed since the task is registered.

```
register         deadline      now
        |--------|
       timeout secs
```

so the time elapsed is now - deadline + timeout